codelab-master/apps/playground/src/app/quiz/shared/models/Quiz.model.ts

```diff
@@ -0,0 +1,10 @@
+import { QuizQuestion } from '@codelab-quiz/shared/models/QuizQuestion.model';
+
+export interface Quiz {
+  quizId: string;
+  milestone: string;
+  summary: string;
+  image: string;
+  questions: QuizQuestion[];
+  status: 'started' | 'continue' | 'completed' | '';
```

Comment:
what's an empty status? Can we drop it?
Can we use enum instead?

Reply:
was using status: '' on the QUIZ_DATA object, that's why there's a '' status. I suppose I can use enum before the class names.

Reply:
Ok, using enums in QuizComponent and ResultsComponent, need to check if it works once I get navigation working...

Reply:
added enum for status, started a quiz and then returned to quiz-selection screen and continue status icon displays on all mat-grid-tiles, not sure why...